### DISCUSSION TITLE
KERI Implementer's Call 06/27/2024

### DISCUSSION THREAD

**Discussion by kentbull** [WebOfTrust/keri#90] [Meetings]
Reports from:
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
WOT-Terms - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

Please add comments ahead of tomorrow's meeting for any additional topics.

**Comment by kentbull** [comment]
## SigTS
No new issues or PRs since last week.


## CESRide
- [docs: IETF to ToIP](https://github.com/WebOfTrust/cesride/pull/169/files) #169 - a one-line docs update to the ToIP spec.
- [chore: Update dependencies](https://github.com/WebOfTrust/cesride/pull/170) #170 - fixes outdated build warning and updates to latest deps.

**Comment by rodolfomiranda** [comment]
## KERIA
### PRs
- https://github.com/WebOfTrust/keria/pull/256
- https://github.com/WebOfTrust/keria/pull/254

### New Issues
- https://github.com/WebOfTrust/keria/issues/257
- https://github.com/WebOfTrust/keria/issues/258
- https://github.com/WebOfTrust/keria/issues/259

**Comment by pfeairheller** [comment]
## SigPy

* No changes

**Comment by pfeairheller** [comment]
## KERIpy

**PRs:**

- https://github.com/WebOfTrust/keripy/pull/806
- https://github.com/WebOfTrust/keripy/pull/798
- https://github.com/WebOfTrust/keripy/pull/796
- https://github.com/WebOfTrust/keripy/pull/793

**Issues**

- https://github.com/WebOfTrust/keripy/issues/672
- https://github.com/WebOfTrust/keripy/issues/807
